### ISSUE TITLE
Selected State does not annonce when Button Component selected

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30954]
## Description
Does not announce "selected" on the focus of the Button Component.

This issue is covered in-depth by parent issue #30842, please check there for more details.

## React Native version:
0.63

**Comment by amarlette** [comment]
Thank you @intergalacticspacehighway for your contribution to improving React Native Accessibility!

**Comment by intergalacticspacehighway** [comment]
Thank you for listing the tasks and detailed explanations! @amarlette 😄

**Comment by huzaifaaak** [comment]
@amarlette PR #31001 should resolve this aswell. Can you please test ?

**Comment by amarlette** [comment]
@huzaifaaak I will follow up with @kacieb on the status of testing for this.

**Comment by kacieb** [comment]
> @amarlette PR #31001 should resolve this aswell. Can you please test ?

I just confirmed by testing on my device that the linked PR resolved this issue! This issue can be closed.

**Comment by amarlette** [comment]
Great thank you for testing!
